mxcube3/ui/components/SampleGrid/TaskItem.jsx

```diff
@@ -158,7 +161,7 @@ export class TaskItem extends React.Component {
       taskStatus = 'Collected';
     }
 
-    return `${task.label}${point} (${taskStatus})`;
+    return `${task.label} (${taskStatus})`;
```

Comment:
a bit picky I know, but why removing the point here?

Reply:
The point was removed because some users said that they thought it might be confusing to display the point number for a already collected sample. That is confusion between points on the mounted sample and already collected ones. Also there were some cases when it was undefined which also looked strange.

So it was removed until we have come up with a better way to display it